src/ace_time/common/compat.h

```diff
@@ -93,6 +93,20 @@
   #include <pgmspace.h>
   #define acetime_strcmp_P strcmp_P
 
+
+
+#elif defined(ARDUINO_ARCH_STM32)
+  #include <avr/pgmspace.h>
+  #define FPSTR(p) (reinterpret_cast<const __FlashStringHelper *>(p))
+
+  // strcmp_P(a,b) is defined to be strcmp(a,b), but we need a function
+  // pointer, so map it directly to strcmp()
+  #define acetime_strcmp_P strcmp
+
+  #undef SERIAL_PORT_MONITOR
+  #define SERIAL_PORT_MONITOR Serial
```

Comment:
Curious, what does the STM32duino define `SERIAL_PORT_MONITOR` to be? My understanding is that it is supposed to be defined to be the serial port that the Arduino IDE uses in its Serial Monitor.

If STM33duino does not define `SERIAL_PORT_MONITOR` at all, I think I would prefer this:
```C++
#ifndef SERIAL_PORT_MONITOR
#define SERIAL_PORT_MONITOR Serial
#endif
```